.travis.yml

```diff
@@ -13,6 +14,8 @@ android:
     - extra-google-m2repository
   licenses:
     - '.+'
+before_install:
+  - chmod +x gradlew
```

Comment:
are these changes required? If so, why?

Reply:
The Travis was failing so I referred to a post and made changes to solve but still the problem was nort solved.